configuration/docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: "3.4"
+
+services:
+  etcd1:
```

Comment:
`1`, is there a `2`?

Reply:
not yet ... but this seems a convention that people use. Happy to remove the indexing if you find it confusing?

Reply:
I think `docker-compose` handles numbering of replicas itself: https://docs.docker.com/compose/compose-file/#replicas

Reply:
@petereast ... problem with that is that the env vars `ETCD..." need changing differently for each instance.

Reply:
this docker-compose configuration is unsuitable for multiple node deployments (or at least we wouldn't use it that way)

Reply:
well its just a name :grin: